### PR TITLE
Updated documentation to add instructions about PLEXOS 9.

### PR DESCRIPTION
**Pull request by k1nshuk**
Updated instructions for setup and running PLEXOS 9 on Eagle.

### CODE REVIEW

**Review by tdthatcher — DISMISSED** [review]

File: applications/plexos-hpc-walkthrough/Run-PLEXOS.md [hunk 1]
```diff
@@ -2,13 +2,61 @@ Please follow the instructions in [`Setup-PLEXOS.md`](Setup-PLEXOS.md) before
...
+```bash
+$PLEXOS/PLEXOS64 -n 5_bus_system_v2.xml -m 2024_yr_15percPV_MT_Gurobi
+```
 
-Currently, the latest available PLEXOS version on Eagle is 8.3R09. Models using
-this version of PLEXOS can work with both XpressMP and Gurobi solvers. However,
-moving forward, PLEXOS support for XpressMP will be dropped in the coming months.
+The command above assumes that we are running the model `2024_yr_15percPV_MT_Gurobi` from file
+`5_bus_system_v2.xml`. PLEXOS 9.0RX requires validating user-credentials for a local 
+PLEXOS account for each run. Therefore, if we ran the above command in an interactive session.
```
Comment (yandthj) [inline applications/plexos-hpc-walkthrough/Run-PLEXOS.md:39]: Minor grammatical error here: this needs to be one sentence or reworded: "Therefore, if we ran the above command in an interactive session. We would need to enter the following username and password"

File: applications/plexos-hpc-walkthrough/Setup-PLEXOS.md [hunk 2]
```diff
@@ -1,13 +1,55 @@
-# Setting up PLEXOS for Eagle
+# Setting up PLEXOS on Eagle
 
-The following file describes the initial setup needed to run PLEXOS on NREL's
-HPC Eagle.
+THe following file contains information about setting up PLEXOS for the first time on your account.
```
Comment (yandthj) [inline applications/plexos-hpc-walkthrough/Setup-PLEXOS.md:3]: Capitalization error

File: applications/plexos-hpc-walkthrough/Setup-PLEXOS.md [hunk 3]
```diff
@@ -1,13 +1,55 @@
...
 
-## Create a new PLEXOS License
+## Setting up the license file
 
 Before we can run PLEXOS, we need to create a license file in eagle. For this,
```
Comment (yandthj) [inline applications/plexos-hpc-walkthrough/Setup-PLEXOS.md:7]: Eagle should be capitalized

**Review by yandthj — CHANGES_REQUESTED** [review]

File: applications/plexos-hpc-walkthrough/Run-PLEXOS.md (hunk 1 above)
Comment (k1nshuk) [reply applications/plexos-hpc-walkthrough/Run-PLEXOS.md]: Removed period, added comma

File: applications/plexos-hpc-walkthrough/Setup-PLEXOS.md (hunk 3 above)
Comment (k1nshuk) [reply applications/plexos-hpc-walkthrough/Setup-PLEXOS.md]: Fixed in the most recent commit

**Review by yandthj — APPROVED** [review]

**Pull request merged**